test/unit/testUtil.js

```diff
@@ -168,13 +172,17 @@ define('testUtil', ['bluebird', 'json!/test/testConfig.json'], (Promise, TestCon
         });
     }
 
+    /**
+     * Clears the global Runtime element and other KBase-used caches.
+     */
     function clearRuntime() {
         if (window.kbaseRuntime) {
             if (window.kbaseRuntime.clock) {
                 window.kbaseRuntime.clock.stop();
             }
             window.kbaseRuntime = null;
         }
+        StagingFileCache.clearCache();
```

Comment:
Added this clear call since the lingering cached calls were causing problems in unexpected tests.